README.md

```diff
@@ -33,6 +33,7 @@ SkyWalking PHP Agent requires SkyWalking 8.4+ and PHP 7.2+
   * [ ] [php-amqp](https://github.com/php-amqp/php-amqp)
   * [ ] [php-rdkafka](https://github.com/arnaud-lb/php-rdkafka)
   * [x] [predis](https://github.com/predis/predis)
+  * [x] [php-amqplib](https://github.com/php-amqplib/php-amqplib)
```

Comment:
Do you think we need to put `producer` here?
```suggestion
  * [x] [php-amqplib](https://github.com/php-amqplib/php-amqplib) for Message Queuing Producer.
```

Reply:
Also, you mentioned amqp here, but the component ID is `rabbitmq-producer`. Is only Rabbit MQ supported?

Reply:
I will update the README and doc later.

Reply:
Not only RabbitMQ supported, but I don't found the component ID about amqp protocol, the library `php-amqplib` is commonly used in message queues services which supports the ampq protocol.

Reply:
If you need a general type for `amqp`, you have to register it here, https://github.com/apache/skywalking/blob/master/oap-server/server-starter/src/main/resources/component-libraries.yml

Also, you may need mapping to indicate it is a queue server(without a specific MQ type), or an icon for topology.